sklearn/metrics/tests/test_common.py

```diff
@@ -1110,3 +1112,97 @@ def test_no_averaging_labels():
             score_labels = metric(y_true, y_pred, labels=labels, average=None)
             score = metric(y_true, y_pred, average=None)
             assert_array_equal(score_labels, score[inverse_labels])
+
+
+def test_metric_permutation_invariance_multiclass():
+    rng = np.random.RandomState(42)
+    y_true = rng.randint(0, 20, 100)
```

Comment:
Completely uniform distribution may lead to uninteresting metric results. You can get a more interesting distribution with:
```
p_C = rng.rand(n_classes)
p_C /= p_C.sum()
cp_C = np.cumsum(p_C)
y_true = np.searchsorted(cp_C, rng.rand(n_samples))
```